sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -546,6 +604,11 @@ export interface TargetSentiment {
     text: string;
 }
 
+// @public
+export interface TextAnalyticsAction {
```

Comment:
do you think it is worth having this with only one value?
i.e. `model-version` is everywhere and the value can be different per action, so should it be here instead of the specific type? Same with `stringIndexType`?
or should this type be removed and put the actionName in each type

Reply:
I added `modelVersion` to the new `TextAnalyticsAction`, thanks for pointing that out! For `stringIndexType`, it does not apply to extract key phrases action so I left it out for now.